bot-twitter/src/index.ts

```diff
@@ -15,7 +16,11 @@ const loop = async function (): Promise<void> {
         clearInterval(refetchIntervalId);
     }
     try {
-        (await getNewAuctions(ETHEREUM_NETWORK)).map(notify);
+        const activeAuctions = await getAllActiveAuctions(ETHEREUM_NETWORK);
+        const newAuctions = getNewAuctionsFromActiveAuctions(activeAuctions);
```

Comment:
No it wouldn't. I changed it so that the newAuctions function does not need to make calls to the blockChain as it takes in the auctions object (activeAuctions). This way we do not need to make two separate calls.